STARTERKIT/package.json

```diff
@@ -9,7 +9,7 @@
     "browser-sync": "^2.12.3",
     "chroma-sass": "^1.2.4",
     "del": "^2.2.0",
-    "eslint": "^2.9.0",
```

Comment:
why eslint removed?

Reply:
https://www.npmjs.com/package/gulp-eslint we have this. otherwice conflict in load plugins.